tests/neural_compressor/test_ipex.py

```diff
@@ -79,5 +80,5 @@ def test_ipex_static_quantization_with_smoothquant(self, task, model_arch, expec
                 is_static=True,
                 num_samples=num_samples,
                 load_inc_model=False,
-                load_ipex_model=True,
+                load_ipex_model=False,
```

Comment:
why not ? my understanding is this test is specifically written to do "INC quant -> IPEX load"

Reply:
The function (IPEX load inc model) is deprecated because IPEXModel is no longer support jit mode, and the model saved by INC is jit mode. And yes, the test name is quiet confused, we will change it.